### ISSUE TITLE
Telegram bot Conflict Error

### ISSUE THREAD

**Issue by Yoken0S** [ytrevor81/TradingView-Binance-Telegram-Bot#7]
Hi,

I got a conflict error with telegram despite I closed every other session, reboot my VPS and change multiple time my bot and so the token.

`telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
`

I have no idea what to do..
Is there a way to use the bot without telegram ?

**Comment by ytrevor81** [comment]
Yes you can comment out or delete all references to telegram in the code, although that would be tedious to do. Admittedly I haven't updated this project at all since I uploaded it to Github and I'm sure the problem resides in outdated code or deprecated telegram libraries.

**Comment by ytrevor81** [comment]
Are you using a telegram bot? I know issues similar to this were resolved in the past by the user creating their own telegram bot

**Comment by Yoken0S** [comment]
When I restart apache the bot works like 5s then crashs. I already tried with new bot and new token but it is the same.
The weirdest part is that the page works cause I have a 405 error (waiting POST data) and not a 500 Error (crash) BUT when I sent "POST" json datas the reply is a 500 error.. no sense..

I'm new with WSGI but maybe it starts several instances or like you said something isn't right with the code.

In a first time I'll try to install it on a raw server dedicated to that to avoid any interferences with my others bots.
And if it is the same I'll be sure that there is a code problem and I'd try to remove the telegram parts but honestly I'm sure if I'll try this.

**Comment by Yoken0S** [comment]
Hello,

I took some time but I instal lthe bot on a full new server dedicated to that and still the same problem...
Like you said the code seems out to date :/

**Comment by Yoken0S** [comment]
When I opened the db.sqlite I saw there were several lines and lot of 0, None

I supposed that there should be only one so I comment line that auto add:  0, ""
And manually enter chat_id and username

And it .. almost..works..
Conflict error always comes but not everytimes and disapears sometimes.

Well but I got another error, it says that my API - IP - Permission isn't correct but I allow all permissions and unsecure the connection to avoid IP problem. And I'm sure that I put the right API / Secret.
